db/migrate/20230512124828_add_status_checked_at_to_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusCheckedAtToProjects < ActiveRecord::Migration[5.2]
+  def change
+    add_column :projects, :status_checked_at, :datetime
```

Comment:
Thoughts about adding an index on this column as well, since we're likely to be needing to order by this column a decent amount to get the oldest rows?